x-pack/plugins/apm/public/components/app/RumDashboard/hooks/useHasRumData.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useFetcher } from '../../../../hooks/use_fetcher';
+
+export function useHasRumData() {
+  const { data } = useFetcher((callApmApi) => {
+    return callApmApi({
+      endpoint: 'GET /api/apm/observability_overview/has_rum_data',
+    });
+  }, []);
+
+  return data;
```

Comment:
I feel like this would be more symmetrical with `useFetcher` if it just returned the result of the `useFetcher` call which would include both `data` and `status`. It looks like you aren't using `status` on rum home but I'd expect something that's a thin wrapper around `useFetcher` to return a similarly shaped result.